tests/MutantTest.php

```diff
@@ -0,0 +1,122 @@
+     */
+    public function getFileGenerator()
+    {
+        $generator = $this->prophesize('Humbug\TestSuite\Mutant\FileGenerator');
+        $generator->generateFile(Argument::type('Humbug\Mutation'))
+            ->willReturn(__DIR__ . '/_files/mutants/mutant.pĥp');
+
+        return $generator->reveal();
+    }
+
+    /**
+     * @param bool $throwException
+     *
+     * @return CoverageData
+     */
+    public function getCoverageData($throwException = false, array $tests = [], array $testMethods = [])
+    {
+        $coverageData = $this->prophesize('Humbug\Utility\CoverageData');
+        $getTestMethod = $coverageData->getTestClasses(Argument::any(), Argument::any());
+
+        if ($throwException) {
+            $getTestMethod->willThrow(new NoCoveringTestsException());
+        } else {
+            $getTestMethod->willReturn($tests);
+        }
+
+        $coverageData->getTestMethods(Argument::any(), Argument::any())
+            ->willReturn($testMethods);
+
+        return $coverageData->reveal();
+    }
+
+    public function getMutation()
+    {
+        return new Mutation(
+            __DIR__ . '/_files/mutables/Math.php',
+            8,
+            'Phpunit_MM1_Math',
+            'add',
+            1,
+            '\Humbug\Mutator\Arithmetic\Addition'
+        );
+    }
+
+    public function testProperties()
+    {
+        $mutation = $this->getMutation();
+        $mutant = new Mutant(
+            $mutation,
+            $this->getFileGenerator(),
+            $this->getCoverageData(false, [ 'dummy' ], [ 'dummyMethod' ]),
+            __DIR__ . '/_files/mutables/'
+        );
+
+        $this->assertEquals($mutation, $mutant->getMutation());
+        $this->assertEquals(__DIR__ . '/_files/mutants/mutant.pĥp', $mutant->getFile());
+        $this->assertEquals(['dummy'], $mutant->getTests());
+    }
+
+    /**
+     * @expectedException \Humbug\Exception\NoCoveringTestsException
+     */
+    public function testConstructorFailsWithoutCoverage()
+    {
+        $mutation = $this->getMutation();
+        $mutant = new Mutant(
+            $mutation,
+            $this->getFileGenerator(),
+            $this->getCoverageData(true),
+            __DIR__ . '/_files/mutables/'
+        );
+    }
+
+    public function testToArray()
+    {
+        $mutation = $this->getMutation();
+        $mutant = new Mutant(
+            $mutation,
+            new FileGenerator(__DIR__ . '/_files/mutants/'),
+            $this->getCoverageData(false, [ 'dummy' ], [ 'dummyMethod' ]),
+            __DIR__ . '/_files/mutables/'
+        );
+
+        $expected = [
+            'file' => 'Math.php',
+            'mutator' => '\Humbug\Mutator\Arithmetic\Addition',
+            'class' => 'Phpunit_MM1_Math',
+            'method' => 'add',
+            'line' => 8,
+            'tests' => [ 'dummyMethod' ]
+        ];
+
+        $actual = $mutant->toArray();
+
+        foreach ($expected as $key => $value) {
```

Comment:
My rationale for testing in a loop against matching the exact array was to allow toArray() to return an array that contains more keys than tested against without failing the tests (regardless of my laziness on the `diff` key)